options/locale/locale_el-GR.ini

```diff
@@ -2482,6 +2482,7 @@ members.invite_now=Πρόσκληση Τώρα
 
 teams.join=Συμμετοχή
 teams.leave=Αποχώρηση
+teams.view=Προβολή
```

Comment:
Don't translate directly. Use crowdin to translate.

Reply:
@lunny Thanks for the feedback. I will remove the translations from the diff